target/classes/templates/start.html

```diff
@@ -5,46 +5,7 @@
-    lager dry bunghole bung Barley bittering, carboy mouthfeel double caramel craft amber attenuation additive de garde.
-    Amber bock rest dunkle adjunct mash weisse pitch conditioning aroma Acidic bottle, stout Adjunct copper squares
-    amber Autolysis lauter Bitter dry kettle, lagering draft units plato dextrin secondary brewing pub Bung brewpub.
-    Units krausen saccharification mead bittering barrel ipa bock adjunct cold, mouthfeel weisse  abv dry craft carboy
-    brewpub priming, berliner Barley acid seidel Becher conditioned real Bacterial.
-</p>
-
-<p>
-    Imperial hopping aroma life Adjunct sparge brew barrel brewing length Barrel, Bitter degrees barleywine dry
-    Barleywine yeast Alpha tank oxidized back All-malt, final beer conditioned bacterial lauter Acidic weisse bitter
-    saccharification. Hoppy acid microbrewery hard scotch noble Anaerobic barleywine Abv imperial, bacterial  rims
-    hops Berliner hand alcohol krug Back ester, cold extract length ale Brew mouthfeel hydrometer acidic. Hop bock
-    Autolysis pump barrel Becher original tank Acidic, secondary life lauter additive units all-malt Additive,
-    Berliner Anaerobic ipa Alpha of dextrin Abv. Shelf squares Alcohol acidic microbrewery craft secondary yeast
-    finishing draught, Aau All-malt pitching noble alpha grainy barley Abv, ale top-fermenting ipa sour specific
-    anaerobic aroma hoppy. Aau hop dextrin Alpha ester draught shelf Attenuation alpha bock imperial Becher original,
-    microbrewery wheat amber Adjunct rims becher conditioned scotch Autolysis abv adjunct, goblet finishing
-    saccharification barrel primary aroma reinheitsgebot brew infusion cider hops.
-</p>
-
-<p>
-    Wort terminal goblet length pub rest Aau fermenting, cider Additive Beer Back krausen ale. Fermenting
-    top-fermenting length pub grainy black becher Amber double Acidic Balthazar dunkle, lambic fermentation sour
-    specific pitching real seidel ale Bitter anaerobic. Heat aerobic acid Amber final finishing double aroma Autolysis
-    acidic back krausen lagering, imperial shelf abv dextrin Additive brewpub Beer adjunct malt amber. Gravity mead
-    Ale microbrewery scotch oxidized length Autolysis bottom finishing imperial hefe cask additive Acid, cold copper
-    sparge all-malt original noble lagering Bung Attenuation back Berliner shelf. Acid anaerobic Acidic pitch becher
-    pump units Bacterial mouthfeel Amber krug real, bright dunkle top-fermenting ipa brewpub reinheitsgebot cask
-    barley glass aroma, Barley wort conditioned tulip goblet pitching double bottle lager craft. Adjunct heat cask
-    beer Aerobic  conditioned krausen, noble sparge units lagering microbrewery bright Berliner tulip, draft Barrel
-    Beer hops scotch ipa.
-</p>
-<span>Lorem generated using Beer Ipsum</span>
+<noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
Something went wrong and it wouldn't work in my intelliJ and it was shutting down frequently.